coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -21,7 +21,7 @@ class DocstyleDefinition:
 
     @enforce_signature
     def __init__(self, language: str, docstyle: str, markers: (Iterable, str),
-                 metadata: Metadata):
+                 metadata: Metadata, padding: (Iterable, str)):
```

Comment:
let me see though. But I don't think that would cause any problem cause `load` function provides an empty tuple if it finds nothing. Which I later break into the properties of `top_padding` and `bottom_padding`. which is later taken care by https://github.com/coala/coala/pull/4557#discussion_r129518147